src/audio_midi.h

```diff
@@ -0,0 +1,188 @@
+/*
+ * This file is part of EasyRPG Player.
+ *
+ * EasyRPG Player is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * EasyRPG Player is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with EasyRPG Player. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef EP_AUDIO_MIDI_H
+#define EP_AUDIO_MIDI_H
+
+// Headers
+#include "audio_decoder.h"
+#include "midisequencer.h"
+
+#define EP_MIDI_FREQ 22050
```

Comment:
Watch out. I know 22050 is less process usage but this preset (at least in my windows x86) is making melodies sounds a bit "uncoordinated" in difference with 44100. 

Here's a little example with Battle3.mid (sorry, github doesn't like audios =(  )
With 44100:
https://drive.google.com/file/d/1ODdgq7aU0tljQwopjRmnyjY-p2kokUv_/view?usp=sharing
With 22050:
https://drive.google.com/file/d/1ti2VP5uFELNgIE7TgTlJJiJuPeV-VO1P/view?usp=sharing

Reply:
Would like to know how much less process usage really is this? Especially on desktop platforms?

I'm all for saving CPU cycles but 22khz is sacrificing audio quality in a big way. I would prefer to default to "reasonable" audio quality that "sounds like RPG_RT or better" and only cut back if we find we really need to on limited hardware.

Also on modern systems, using low frequencies like this for music is very uncommon and so downstream software / OS audio stacks may up-sample it poorly.

For all these conjectures about processing costs of audio features, we really need some benchmarks and performance numbers. Some of them could be costing much less or more more CPU than we think.

That being said, I also really want to see some initial version of this merged to master so I can being playing with it while testing other PRs. So I'm fine collecting the outstanding questions into a github issue, getting this into a base working state, and merging it.